aten/src/ATen/native/cuda/Embedding.cu

```diff
@@ -332,6 +332,8 @@ Tensor embedding_dense_backward_cuda(const Tensor & grad_, const Tensor & indice
 
 Tensor & embedding_renorm_cuda_(Tensor & self, const Tensor & indices,
                                 double max_norm, double norm_type) {
+  // See Note [Writing Nondeterministic Operations]
```

Comment:
Also needs a comment explaining why the operation is nondeterministic

Reply:
Right, I'm not sure why it's nondeterministic yet. I asked Natalia back in the issue if she knows.